wagtail/admin/templates/wagtailadmin/edit_handlers/inline_panel.html

```diff
@@ -24,8 +24,8 @@
 </script>
 
 <p class="add">
-    <a class="button bicolor button--icon" id="id_{{ self.formset.prefix }}-ADD">
+    <button class="button bicolor button--icon" id="id_{{ self.formset.prefix }}-ADD">
```

Comment:
suggestion: use

```suggestion
    <button type="button" class="button bicolor button--icon" id="id_{{ self.formset.prefix }}-ADD">
```

otherwise pressing enter anywhere on the form will trigger a submit. See #6592 for context

Reply:
Somehow my second half of the comment got lost somwhere -- just wanted to say thank you for this seemingly small change, but nonetheless quite important for making the Wagtail admin more accessible

Reply:
Oops! I entirely forgot about that, good catch. Also makes any change to expandind_formset.js unnecessary. Thanks 😄